### PR TITLE
Feat/create new command request point

### PR DESCRIPTION
**Pull request by YoelFerreyra**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -6,6 +6,8 @@ ADD_POINT_TAG_ID = 1258801833191669802 # ID for task completed tag.
 ADD_BOOSTED_POINT_TAG_ID = 1263873487953592381 # ID for the tag that adds boosted points
 DISCORD_QA_ROLE_ID=1140734540139204800
 DISCORD_QA_CHANNEL_ID=qa-request
+ADMINISTRATOR_TAG_ID=1140734540139204800
+ADMIN_POINT_REQUEST_CHANNEL=1270770663048613983
```
Comment (heliomar-pena) [inline .env.example:10]:
```suggestion
ADMINISTRATOR_TAG_ID=1014267063788908585
ADMIN_POINT_REQUEST_CHANNEL=1310614733803618416
```

Comment (heliomar-pena) [reply .env.example:10]: Adding as default the ones that we have on our prod server

**Review by heliomar-pena — APPROVED** [review]
Good work! Just change the .env examples for the suggested

File: .env.example (hunk 1 above)
Comment (YoelFerreyra) [reply .env.example]: Complete

**Review by heliomar-pena — DISMISSED** [review]

File: src/commands/utility/requestPoint.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+          ephemeral: true,
+        });
+      }
+
+      const user = options.getUser('user');
+      const userMessage = options.getString('reason');
+      const escapedUserId = `<@${user.id}>`;
+
+      const threadLink = `https://discord.com/channels/${guild.id}/${channel.id}`;
+      const message = `${escapedUserId} está pidiendo un punto por **${userMessage}**. Revisa el mensaje acá: ${threadLink}`;
```
Comment (Israel-Laguan) [inline src/commands/utility/requestPoint.js:36]: Change to English, please

Comment (YoelFerreyra) [reply src/commands/utility/requestPoint.js]: Complete

**Review by heliomar-pena — APPROVED** [review]
Good work!

**Pull request merged**